src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/PodTemplateStepExecution.java

```diff
@@ -56,6 +56,7 @@ public boolean start() throws Exception {
         newTemplate.setContainers(step.getContainers());
         newTemplate.setNodeSelector(step.getNodeSelector());
         newTemplate.setServiceAccount(step.getServiceAccount());
+        newTemplate.setAnnotations(step.getAnnotations());
```

Comment:
I was unaware of that. I just did a test where I upgraded from 0.7, to my version, and all the data remained.

I believe it worked because I declare `annotations` here with an initial value: https://github.com/jenkinsci/kubernetes-plugin/pull/105/files#diff-37ffb9809d7b186d5733b266c664ba41R36